config/rbac/role.yaml

```diff
@@ -79,6 +79,21 @@ rules:
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
+metadata:
+  name: manager-role
+  namespace: openshift-config
```

Comment:
so per your comment here:

https://github.com/openshift/lightspeed-operator/pull/153#issuecomment-2162690983

does this role need to be turned into a clusterrole?

Reply:
that's generated by controller-gen. 
If we remove the namespace limit [here](https://github.com/openshift/lightspeed-operator/pull/153/files#diff-67e7ce7f3b59749b56b35a828cd9e81ce48fcda60db9821752fe3f8b04780051R81-R82), these permissions will be added to the clusterrole of the operator service account. 

But for some reason yet to find out with the OLM team, the operator service account has already access to configmaps in other namespaces due to cluster role created during the bundle installation.